### ISSUE TITLE
koto-boilerplate vs koto-starter?

### ISSUE THREAD

**Issue by ialarmedalien** [kotojs/koto-boilerplate#1]
This is a question rather than a bug/issue -- I started using `koto-starter` for a project, but encountered various issues that meant I couldn't use it out of the box (node packages missing, incorrect capitalisation of `koto`, etc.). I've just discovered this repo, which does work without any fiddling (and uses webpack, too!). Has `koto-starter` been superseded by `koto-boilerplate`?

**Comment by nicksrandall** [comment]
Yes, koto-starter is built on some technology that is not supported anymore. I'll go ahead and add a comment to that repo stating that it has been deprecated and redirect users to this repo. Sound good?
